enhanced-date.js

```diff
@@ -1 +1,101 @@
-// export this as a Node module using the code from Module_2
\ No newline at end of file
+// export this as a Node module using the code from Module_2
+'use strict';
+
+var dateObj = (function() {
```

Comment:
Don't use the immediately invoked function expression in the node module. Instead, removed everything from it, and change what's being returned to what module.exports is set to.